### PR TITLE
Removed Site Owners from the response

### PR DESCRIPTION
**Pull request by YuriySamorodov**
Graph API does not return site owners as of now

### CODE REVIEW

**Comment by Lauragra** [comment]
@JeremyKelley , please review and approve. It looks like there is a validation error on the JSON in this PR.

**Comment by YuriySamorodov** [comment]
@Lauragra , could you please review this PR?

**Comment by YuriySamorodov** [comment]
@JeremyKelley , could you please have a look?

File: api-reference/beta/api/site-get.md [hunk 1]
```diff
@@ -95,17 +95,14 @@ Content-type: application/json
...
-  },
   "displayName": "OneDrive Team Site",
   "name": "1drvteam",
   "createdDateTime": "2017-05-09T20:56:00Z",
   "lastModifiedDateTime": "2017-05-09T20:56:01Z",
-  "webUrl": "https://contoso.sharepoint.com/teams/1drvteam"
+  "webUrl": "https://contoso.sharepoint.com/teams/1drvteam",
+  "root": "",
+  "description": "",
+  "siteCollection": @{hostname=https://contoso.sharepoint.com}
```
Comment (Lauragra) [inline api-reference/beta/api/site-get.md:105]: There's a JSON validation error here - please fix.

**Review by Lauragra — COMMENTED** [review]
Please fix the JSON so the PR passes validation.

File: api-reference/beta/api/site-get.md (hunk 1 above)
Comment (YuriySamorodov) [reply api-reference/beta/api/site-get.md]: fixed

**Review by YuriySamorodov — COMMENTED** [review]
fixed

**Pull request closed without merging**